### PR TITLE
Update jenkins-stability-checklist.md

### PR DESCRIPTION
**Pull request by ThomasLehnerSpryker**
removed non supported checklist boxes
fixed non working image reference

## PR Description

TBD

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/ca/dev/best-practices/jenkins-stability-checklist.md [hunk 1]
```diff
@@ -7,46 +7,53 @@ redirect_from:
...
-- [ ] Profile your jobs locally to understand their normal memory demand, especially when interacting with data.
-- [ ] In a standard-sized non-production environment, run lengthy imports and sync processes lasting more than 1-2 hours.
-- [ ] Refer to troubleshooting instructions for further assistance.
-- [ ] Be prepared to lose manually created jobs; ensure that all critical jobs are persisted in your project (jenkins.php).
+This checklist is designed to help you implement Spryker’s best practices, to enhance the stability and performance of the Jenkins component in your Spryker PaaS environment. Before raising issues about Jenkins performance and stability with Spryker, make sure you have fully completed the checklist. If you have concerns or questions about it, raise them with Spryker Support.
+
+
+- Configure a maximum of two executors.
+- Set your PHP `memory_limit` value to be less than 2 GB.
+- Implement batch processing (guidelines are here and here) in your importers and be mindful of the maximum memory consumption.
```
Comment (lenadoc) [inline docs/ca/dev/best-practices/jenkins-stability-checklist.md:15]: @ThomasLehnerSpryker please add the links to guidelines

File: docs/ca/dev/best-practices/jenkins-stability-checklist.md [hunk 2]
```diff
@@ -7,46 +7,53 @@ redirect_from:
...
+- Fine-tune the chunk size of the queues you work with.
+- Make sure that your theoretical maximum memory demand for all planned parallel processes remains below the memory allocation of your Jenkins instance.
+- Verify that every PHP job you run consumes less memory than your specified PHP memory limit. There shouldn't be the error “PHP Fatal error: Out of memory”.
+- Make sure that no jobs are configured with a non-default memory limit or without any memory limit at all in `jenkins.php`. For example,`php -d memory_limit=-1 vendor/bin/console ...`.
+- Avoid spawning an excessive number of worker. There should be no more than two workers per queue.
+- Profile your jobs locally to understand their normal memory demand, especially when interacting with data.
+- In a standard-sized non-production environment, run lengthy imports and sync processes lasting more than 1-2 hours.
+- Refer to troubleshooting instructions for further assistance.
```
Comment (lenadoc) [inline docs/ca/dev/best-practices/jenkins-stability-checklist.md:23]: What troubleshooting instructions do you mean here?

**Review by lenadoc — CHANGES_REQUESTED** [review]

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/ca/dev/best-practices/jenkins-stability-checklist.md (hunk 1 above)
Comment (ThomasLehnerSpryker) [reply docs/ca/dev/best-practices/jenkins-stability-checklist.md]: https://docs.spryker.com/docs/dg/dev/data-import/202311.0/data-import-optimization-guidelines.html
https://docs.spryker.com/docs/dg/dev/integrate-and-configure/integrate-elastic-computing.html

File: docs/ca/dev/best-practices/jenkins-stability-checklist.md (hunk 2 above)
Comment (ThomasLehnerSpryker) [reply docs/ca/dev/best-practices/jenkins-stability-checklist.md:23]: Can be removed, the sentence before should read "do not run"

File: docs/ca/dev/best-practices/jenkins-stability-checklist.md [hunk 3]
```diff
@@ -7,56 +7,73 @@ redirect_from:
...
-- [ ] Profile your jobs locally to understand their normal memory demand, especially when interacting with data.
-- [ ] In a standard-sized non-production environment, run lengthy imports and sync processes lasting more than 1-2 hours.
-- [ ] Refer to troubleshooting instructions for further assistance.
-- [ ] Be prepared to lose manually created jobs; ensure that all critical jobs are persisted in your project (jenkins.php).
+This checklist is designed to help you implement Spryker’s best practices to enhance the stability and performance of the Jenkins component in your Spryker PaaS environment. Before raising issues about Jenkins performance and stability with Spryker, make sure you have fully completed the checklist. If you have concerns or questions about it, raise them with Spryker Support.
+
+
+- Configure a maximum of two executors.
+- Set your PHP `memory_limit` value to be less than 2 GB.
+- Implement batch processing (guidelines are here and here) in your importers and be mindful of maximum memory consumption.
```
[the remark this answers is not in the document]
Comment (ThomasLehnerSpryker) [reply docs/ca/dev/best-practices/jenkins-stability-checklist.md:15]: https://docs.spryker.com/docs/dg/dev/data-import/202311.0/data-import-optimization-guidelines.html
https://docs.spryker.com/docs/dg/dev/integrate-and-configure/integrate-elastic-computing.html

**Pull request merged**